content/graph/MaximumClique.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * Author: chilli, Haobin Ni
+ * Date: 2019-05-10
+ * License: CC0
+ * Source: https://en.wikipedia.org/wiki/MaxCliqueDyn_maximum_clique_algorithm,
+ * https://github.com/FTRobbin/Dreadnought-Standard-Code-Library/blob/11c861e55a73be9c32a799bfe4398e0c62c2da15/improve/MaximumClique.cpp
+ * Description: Finds the maximum clique of a graph. Can be used to find the
+ * maximum independent set by finding the clique of the complement graph.
+ * Time: Runs in about ~1s for n=155 and worst case random graphs (p=.95). Runs
+ * faster for sparse graphs.
+ * Status: fuzz-tested
+ */
+struct Maxclique {
+	double limit=0.025, pk=0;
+	struct Vertex { int i, d; Vertex(int i) : i(i), d(0) {} };
+	typedef vector<Vertex> vv;
+	vector<B> e;
+	vv V;
+	vector<vi> C;
+	vi qmax, q, S, So;
+	bool cut1(const int pi, const vi &A) {
+		trav(i, A) if (e[pi][i]) return true;
+		return false;
+	}
+	void init(vv &r) {
+		trav(v,r) v.d = 0;
+		trav(v, r) trav(j, r) v.d += e[v.i][j.i];
+		sort(all(r), [](auto a, auto b) { return a.d > b.d; });
+	}
+	void expand(vv &R, int lev = 1) {
+		S[lev] += S[lev - 1] - So[lev];
+		So[lev] = S[lev - 1];
+		while (sz(R)) {
+			if (sz(q) + R.back().d <= sz(qmax)) return;
+			q.push_back(R.back().i);
+			vv Rp;
```

Comment:
`R`/`Rp` and `S`/`So` is easy to typo, and the mixed casing makes it hard to type. Rename to some other random capital letter?

Reply:
I mean, `S` and `So` stand for `S` and `S old`, so I think they should at least start with the same name.

Reply:
`old`? or some two consecutive letters?